### ISSUE TITLE
Change element signature. 

### ISSUE THREAD

**Issue by MattMcFarland** [trainyard/choo-cli#3]
Element signature is currently `(state, prev, send)`.
Should maybe start as empty or `(props)`

**Comment by yoshuawuyts** [comment]
I reckon empty might make most sense - passing down empty objects creates more strain on the GC + makes it tempting to just chuck down the whole `state` object.

Also perhaps something worth doing is using `require('bel')` rather than `require('choo/html')` so it's easier to refactor out of the application - but that's debatable I think

**Comment by MattMcFarland** [comment]
I think using bel could be confusing, how about just using an empty signature?

**Comment by ronnross** [comment]
That is an interesting concept using `bel` for elements instead of `choo/html`. I like the idea of being able to easily refactor elements out of the app, if need be. 

I wonder if there is a way to use a plain template literal and delay tagging until you render it in your view?

**Comment by ronnross** [comment]
What if we used `const html = require('bel')`? Then a template can be tagged with `html`. Does that seem like it would cut down on confusion, and keep it easy to refactor out of the app?

**Comment by MattMcFarland** [comment]
Partially fixed via commit eecd4c9 - we still would like to keep this open to discussion regarding the template library.

**Comment by MattMcFarland** [comment]
@yoshuawuyts we've updated it again with 9b00d49, example below:

```
choo generate helloWorld
```

Generates `elements/hello-world.js` with the following content:

``` javascript
// Element: helloWorld 
//
// We can use bel instead of choo/html to keeep elements modular
// and allow them to easily move outisde of the app.
const html = require('bel')

function helloWorld () {
  return html`<div> helloWorld </div>`
}

module.exports = helloWorld
```

Thoughts???
